ci/base.yml

```diff
@@ -30,8 +30,8 @@ local_conf_header:
     INHERIT += "buildhistory"
     INHERIT += "rm_work"
   qcomflash: |
-    IMAGE_CLASSES += "image_types_qcom"
-    IMAGE_FSTYPES += "qcomflash"
+    IMAGE_CLASSES += "image_types_qcom image_types_qcom_pkg"
```

Comment:
again, I haven't done a full review, and it's too late to do it in 2024 ;) but I think it would be better with a single class and fstype.